### ISSUE TITLE
No tray icon change or red asterisk for Discord tabs (even though notifications work fine)

### ISSUE THREAD

**Issue by echosa** [ramboxapp/community-edition#1730]
### Steps to reproduce

1. Add Discord to Rambox.
2. Enable Discord notifications.
3. Select a different tab in Rambox and switch to a different application.
4. Get a Discord notification.

### Expected behavior

The tray icon should turn red and the Discord tab in Rambox should get a red asterisk (like it does for other services).

### Actual behavior

Neither of these things happen, even though notifications themselves popup.

### ENVIRONMENT

> Rambox: v0.5.17
> OS: Pop!_OS (Ubuntu variant from System 76)
> Arch: x64

From `Help -> About`:

```
Version: 0.5.17
Platform: linux (x64)
Electron: 1.8.4
Chromium: 59.0.3071.115
Node: 8.2.1
```

**Comment by maddo** [comment]
Same behavior on macOS:

```
Version: 0.5.17
Platform: darwin (x64)
Electron: 1.8.4
Chromium: 59.0.3071.115
Node: 8.2.1
```

**Comment by StormPooper** [comment]
Same on Windows (two machines, here's one of them):
```
Version: 0.5.17
Platform: win32 (x64)
Electron: 1.8.4
Chromium: 59.0.3071.115
Node: 8.2.1
```

**Comment by oma84** [comment]
Should have been fixed with this #1619 , but I assume it doesn't work because of changed class names.
